pkg/agent/svid/rotator.go

```diff
@@ -104,6 +118,8 @@ func (r *rotator) rotateSVID(ctx context.Context) (err error) {
 
 	// Get the mtx before starting the rotation
 	// In this way, the client do not create new connections until the new SVID is received
+	r.c.Log.Debug("Acquiring rotation lock")
+	defer r.c.Log.Debug("Releasing rotation lock")
```

Comment:
This feels excessive - perhaps it's left over from testing?

Reply:
It was helpful in tracking this down, but I can see how it might be too verbose. It's too bad our logging facility doesn't have a trace level.

I'll get rid of it.